src/Microsoft.ML.FastTree/FastTree.cs

```diff
@@ -900,12 +899,12 @@ private double[] GetInitScores(Dataset set)
 
     internal abstract class DataConverter
     {
-        protected readonly int NumFeatures;
+        private protected readonly int NumFeatures;
```

Comment:
>private [](start = 8, length = 7)

For an internal class, is there a difference between protected and private protected?

Reply:
probably not but I don't see this change doing any harm either :-)

---
In reply to: [258221426](https://github.com/dotnet/machinelearning/pull/2511#discussion_r258221426) [](ancestors = 258221426)